{"event": "verbose", "uuid": "5d471746-7ff0-4719-86f9-51bd1153ff47", "counter": 3, "stdout": "\u001b[0;31mThe error appears to be in '/home/vescana/test_ansible_molecule/roles/web_servers/tasks/main.yml': line 29, column 7, but may\u001b[0m", "start_line": 2, "end_line": 3, "runner_ident": "931ea692-9f31-448c-a9c9-7c90d61531db", "created": "2022-04-11T16:22:48.954920"}